.circleci/config.yml

```diff
@@ -124,6 +115,19 @@ commands:
           name: Save test results
           path: ~/TestResults/ResultBundle.zip
           destination: results
+  codegen_test_configurations_generate_and_test:
+    steps:
+      - run:
+          command: git clone -b "$CIRCLE_BRANCH" "$CIRCLE_REPOSITORY_URL"
```

Comment:
This job manually clones the repo because the `checkout` step provided by CircleCI, which we use in other jobs, clones the repo into `/Users/distiller/project` and SPM does not like that for a local package dependency. Manually cloning the repo now places it into `/Users/distiller/project/apollo-ios` and we're able to use local packages as expected.